src/kayenta/canary.dataSource.ts

```diff
@@ -122,7 +121,6 @@ module(CANARY_DATA_SOURCE, []).run([
       lazy: true,
       autoActivate: true,
       defaultData: [],
-      iconName: 'spMenuCanaryReport'
```

Comment:
doesn't the upgraded `core` version mean you can add these?

Reply:
yes I can! Just want to check the PR build first since locally it looks good.